app/controllers/items/index.js

```diff
@@ -4,9 +4,10 @@ import searchModule from "../search_module";
 
 export default searchModule.extend({
 
-  queryParams: ['searchInput', "itemSetId"],
+  queryParams: ['searchInput', "itemSetId", "showQuantityItems"],
```

Comment:
This works well for the single case. How would we make this work at the generalised level? Say if there were cases where we wanted to add a global flag? Wouldn't want to have to add this into every controller that needed it. Is there a general way to add a whitelisted query string argument to the `Ember.Controller` base class?